aosp_diff/preliminary/system/sepolicy/10_0010-0001-Allow-adbd-to-access-proc-net-tcp-tcp6-udp-udp6-.patch

```diff
@@ -0,0 +1,50 @@
+From 5d4e4deb4bf4e5b198c57ed8e8a0dd8a148737f1 Mon Sep 17 00:00:00 2001
+From: Gregory Montoir <gmontoir@google.com>
+Date: Mon, 6 Sep 2021 09:25:13 +0800
+Subject: [PATCH] Allow adbd to access /proc/net/{tcp,tcp6,udp,udp6}
+
+File accesses go through com.android.ddmlib.SyncService for CTS
+ListeningPortsTest.
+
+Bug: 201645790
+Test: atest ListeningPortsTest
+Ignore-AOSP-First: Fix already in AOSP
+Change-Id: I0c66fb5e35cda3b1799cf003402e454d7a951e96
+(cherry picked from commit 1a6b37d838459b3d46cc06e31bc3c2bf5e7c387f)
+Merged-In:I0c66fb5e35cda3b1799cf003402e454d7a951e96
+---
+ prebuilts/api/30.0/private/adbd.te | 3 +++
+ private/adbd.te                    | 3 +++
+ 2 files changed, 6 insertions(+)
```

Comment:
This is also an already applied patch. Please cross-check and remove.
Project ./system/sepolicy
             Already applied         10_0010-0001-Allow-adbd-to-access-proc-net-tcp-tcp6-udp-udp6-.patch

Also the naming convention is wrong